test/unit_test/engine/test_engine.py

```diff
@@ -74,6 +75,25 @@ def test_register_no_search(self):
         assert name in engine.passes
         assert name in engine.pass_order
 
+        # clean
```

Comment:
Hi, instead of these additional tests, Xiaoyu's comment meant some tests like those added in this PR https://github.com/microsoft/Olive/pull/184. 

The current changes to the tests in this PR are not required since these are tests for the engine.

Reply:
Sorry, I misunderstood. Please review the new unit tests.